File: 2678 number-of-senior-citizens/solution.go
func countSeniors(details []string) int {
    ans:=0
    for _,v := range details {
        sub:= v[11:13]
        age,_:= strconv.Atoi(sub)
        if age > 60 {
            ans++
        }
    }
    return ans
}

File: 2678 number-of-senior-citizens/solution.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int ans = 0;
        for(string s:details) {
            string sub = s.substr(11,2);
            int age = stoi(sub);
            if(age > 60) ans++;
        }
        return ans;
    }
};
